s3_website.yml

```diff
@@ -43,7 +43,6 @@ cloudfront_distribution_config:
 # concurrency_level: 5
 
 redirects:
-  articles/can-multi-year-ssl-certificate: /articles/can-multi-year-ssl-certificates/
   articles/secondary-dns-support: /articles/secondary-dns/
```

Comment:
It happened to me in the past with a bugus/outdated version of s3_website. It is possible that whoever deployed lately had an outdated version.